etc/workflows/studio-upload.xml

```diff
@@ -214,72 +216,74 @@
       </configurations>
     </operation>
 
-    <!-- Encode to engage player preview images -->
+    <!-- Encode to engage search result thumbnails -->
 
     <operation
-        id="image"
-        exception-handler-workflow="partial-error"
-        description="Creating player preview image">
+      id="image"
+      exception-handler-workflow="partial-error"
+      description="Creating search result default thumbnails">
       <configurations>
-        <configuration key="source-flavor">*/prepared</configuration>
-        <configuration key="target-flavor">*/player+preview</configuration>
+        <configuration key="source-flavors">*/prepared</configuration>
+        <configuration key="target-flavor">*/search+preview</configuration>
         <configuration key="target-tags">engage-download</configuration>
-        <configuration key="encoding-profile">player-preview.http</configuration>
+        <configuration key="encoding-profile">search-cover.http</configuration>
         <configuration key="time">1</configuration>
       </configurations>
     </operation>
 
-    <!-- Encode to feed cover images -->
+    <!-- Encode to engage player preview images -->
 
     <operation
         id="image"
         exception-handler-workflow="partial-error"
-        description="Creating feed cover image">
+        description="Creating player preview image">
       <configurations>
-        <configuration key="source-flavor">*/prepared</configuration>
-        <configuration key="source-tags"></configuration>
-        <configuration key="target-flavor">*/feed+preview</configuration>
-        <configuration key="target-tags">atom,rss</configuration>
-        <configuration key="encoding-profile">feed-cover.http</configuration>
+        <configuration key="source-flavors">*/prepared</configuration>
+        <configuration key="target-flavor">*/player+preview</configuration>
+        <configuration key="target-tags">engage-download</configuration>
+        <configuration key="encoding-profile">player-preview.http</configuration>
         <configuration key="time">1</configuration>
       </configurations>
     </operation>
 
-    <!-- Create a cover image with the default template -->
-
     <operation
         id="tag"
-        description="Removing unneeded presenter-cover from download publication">
+        exception-handler-workflow="partial-error"
+        description="Tagging source material for archival">
```

Comment:
That's not what this is doing but more than that, why not create the correct flavor in the first place? It doesn't really make sense to move it just after creating it.

Reply:
This was my first intention, too. But extracting the preview images for both tracks in one operation may speedup the processing. It's your choice, should I implement your request?

Reply:
Implemented your suggestion.